src/main/java/core/basesyntax/service/CsvFileHandler.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import com.opencsv.CSVReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileHandler {
+    public List<String[]> processFile(String csvInputFile) {
+        List<String[]> listOfCommands = new ArrayList<>();
```

Comment:
```suggestion
        List<Command/Transaction> listOfCommands = new ArrayList<>();
```
use model instead of String[]